spec/rails_helper.rb

```diff
@@ -1,54 +1,48 @@
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
 require_relative '../config/environment'
-abort("The Rails environment is running in production mode!") if Rails.env.production?
+abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
-require "capybara/rspec"
+require 'capybara/rspec'
 
 begin
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you other **tests** to pass(i.e. all tests of your app `rspec spec`) **even if they are not part of this milestone** to make your app more professional. Below is the result of all your tests(`rspec spec`):
![Screenshot from 2022-11-02 02-16-22](https://user-images.githubusercontent.com/83042742/199365925-a1d787cb-cb39-43c5-a29f-5fb6cbe8d8d1.png)

Reply:
@tresorsawasawa , tests are now passing